### ISSUE TITLE
[Mod Compat]: Name of Resource Creeper in The One Probe not showing up properly

### ISSUE THREAD

**Issue by Raidobw2** [CreeperHost/ResoursefulCreepers#13]
### Modlist
![modlist_with_top](https://user-images.githubusercontent.com/17010632/165395095-6d108b4b-6d19-4448-aee9-0c4ecdcdfd16.png)

### Description
If you have The One Probe installed, the shown name appears to be the language key instead of the actual human readable name of the Resource Creeper.
### Media

https://user-images.githubusercontent.com/17010632/165395212-fefa73e3-e271-4174-a145-9b924893f161.mp4

![creeper_vs_resource_name](https://user-images.githubusercontent.com/17010632/165395250-09e8d7e1-6916-4163-9a6d-3639eac6ec11.png)

**Comment by gigabit101** [comment]
Can't really do localization when everything is dynamically generated, We did used to set the display name on the client and the server but then InControl could not control the spawns, Not really anything I can about this one

**Comment by Raidobw2** [comment]
Hmm I see what you mean. In that case you could add some for the vanilla resource creepers, since they are part of the base game. Modpack makers will likely use the already generated lang keys for those, then you can just ship a notice that other creepers from other mods need to have a translation key added for each custom creeper.
![image](https://user-images.githubusercontent.com/17010632/165651783-0c7d631d-b16a-4ca6-9758-1fd28bb81145.png)
Possibly use the format your code already generates, the id I mean:
```
{
    "entity.resourcefulcreepers.redstone_ore": "Redstone localization!"
}
```
And if you need an example, you can add a "Wiki" tab on CurseForge and I can make you a txt file and an example datapack as well, then you could copy those examples onto your github stuff. Docs are really useful!

If you don't want, then sadge but go ahead and close this issue :P KubeJS/Datapacks can override those lang keys without an issue and work well as additional modpack configuration, which this mod already requires when part of a structured modpack anyways, so it's not a chore, but just needs to be documented imo

**Comment by gigabit101** [comment]
![image](https://user-images.githubusercontent.com/5615438/165750763-f2a542f6-f5f5-4c67-9c65-8ae9fe8d287c.png)

**Comment by gigabit101** [comment]
Auto-generated and added to the resource packs folder, You will just need to enable

**Comment by Raidobw2** [comment]
It unfortunately doesn't work, here is the corrected version. This can be hardcoded in case you need to, the base game doesn't change (neither should your id generation methods) and the resource pack you made is optional in case of problems
```
{
  "entity.resourcefulcreepers.copper_ore": "Copper Creeper",
  "entity.resourcefulcreepers.lapis_ore": "Lapis Creeper",
  "entity.resourcefulcreepers.quartz_ore_creeper": "Quartz Creeper",
  "entity.resourcefulcreepers.gold_ore": "Gold Creeper",
  "entity.resourcefulcreepers.coal_ore": "Coal Creeper",
  "entity.resourcefulcreepers.redstone_ore": "Redstone Creeper",
  "entity.resourcefulcreepers.emerald_ore": "Emerald Creeper",
  "entity.resourcefulcreepers.iron_ore": "Iron Creeper",
  "entity.resourcefulcreepers.diamond_ore": "Diamond Creeper"
}
```

**Comment by Raidobw2** [comment]
![image](https://user-images.githubusercontent.com/17010632/165837391-8407be0a-04cd-4f34-b296-8b12dfcbfc36.png)

**Comment by gigabit101** [comment]
The resource pack should be fixed now. You will need to remove it to allow it to regenerate

**Comment by Raidobw2** [comment]
I've deleted all the Resource Packs in the resourcepacks folder (folder wipe):
![image](https://user-images.githubusercontent.com/17010632/166066894-16c4c180-7504-49a0-ab66-5af7d09db79f.png)
Here it regenerated just now using 1.0.9 (done it twice):
![image](https://user-images.githubusercontent.com/17010632/166066627-1bc7824a-30c0-44d3-bafc-5852161bf422.png)
![image](https://user-images.githubusercontent.com/17010632/166067762-2d824fff-be1a-4bea-9b7c-fd6630de0e7f.png)
But inside it still shows:
![image](https://user-images.githubusercontent.com/17010632/166066662-0dd8006b-95a0-4292-818d-376d0fbd9e69.png)
I've done this twice, but twice it regenerated the same way and doublecked I'm using 1.0.9

**Comment by gigabit101** [comment]
Try deleting the ResourcefulCreepers folder under the config folder as well
as the resource pack and try again, might have failed to cleanup

**Comment by Raidobw2** [comment]
Ok yes it generated in the correct format now. The only issue is the path `resourcepacks\ResourcefulCreepers\assets\resourcefulcreepers\lang` doesn't work for the lang file, like it's not recognized. But if you do `resourcepacks\ResourcefulCreepers\assets\minecraft\lang` (like I had previously done in the KubeJS example actually), it works. I don't know why, but it just works when namespace is minecraft

**Comment by gigabit101** [comment]
Fixed in 1.0.12

**Comment by Raidobw2** [comment]
It's kind of odd but unless I extract the resource pack and use the unzipped one it doesn't work. I don't know why. Might be a MC bug at that point. It works with the unzipped pack, so I'm fine with it :)
![image](https://user-images.githubusercontent.com/17010632/166094986-916ece1d-06ac-439c-9555-eca0bea16c2b.png)
